samples/subsys/shell/adc/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _shell-adc-sample:
+
+Analog to Digital Converter (ADC) shell example
+###############################################
+
+Overview
+********
+
+This example provides shell access to aquiring ADC samples
+
+Requirements
+************
+
+A board with ADC support
+
+Building
+********
+
```

Comment:
OK, I get it about not requiring a console.  If I read things right, the default Zephyr settings will use a serial console for shell I/O, so for this sample to work (as written) don't you need to tell the user to connect your development system to the board via the serial port using something such as putty?  Otherwise, where will the shell prompt show up?

Reply:
@dbkinder Well, yes. The user would of course need to establish contact to the shell in order to use the shell. However, none of the existing shell samples seem to include instructions for how to do this (perhaps because the instructions are specific to the selected shell backend?). Are you aware of a sample with proper instructions which I can use as inspiration?

Reply:
... and we should go back through the shell samples and fix this there too :)